### DISCUSSION TITLE
Searchlight view range should upgrade along with Periscope view range

### DISCUSSION THREAD

**Discussion by Joel-B-Williams** [FakeFishGames/Barotrauma#11870] [Ideas and suggestions]
Searchlights are meant to scope out targets visually - and at T1 they work well.  They have a longer view range than turrets, giving them an interesting use-case

Fairly quickly into a campaign, Turret visual range can be upgraded to surpass the view range of a Searchlight - which combined with their slower turn speed makes them quickly moot

I would suggest either a separate upgrade be made available to increase the view range of Searchlights, or that they scale with the same upgrade to Turrets - keeping them in a useful spot for exploration & scouting

**Comment by MarkCringan** [comment]
Additionally, if you were to use a submarine with no weapons and only searchlights/cameras you don't have the option to upgrade range.

Also camera range upgrade would be nice too!
